ui/ts/pages/nodes.ts

```diff
@@ -37,14 +40,14 @@ module AdminViews {
         private static comparisonColumns: Table.TableColumn<NodeStatus>[] = [
           {
             title: "Node ID",
-            view: (status: NodeStatus): _mithril.MithrilVirtualElement =>
+            view: (status: NodeStatus): MithrilElement =>
               m("a", {href: "/nodes/" + status.desc.node_id, config: m.route}, status.desc.node_id.toString()),
             sortable: true,
             sortValue: (status: NodeStatus): number => status.desc.node_id
           },
           {
             title: "Address",
-            view: (status: NodeStatus): string => status.desc.address.address,
+            view: (status: NodeStatus): string => status.desc.address.string,
```

Comment:
Scratch that: I just changed the proto itself on the backend so the field is "address" again.